yjit/src/asm/mod.rs

```diff
@@ -108,6 +112,21 @@ impl CodeBlock
         let mut dummy_block = vec![0; mem_size];
         let mem_ptr = dummy_block.as_mut_ptr();
 
+        #[cfg(not(feature="asm_comments"))]
+        return Self {
+            dummy_block: dummy_block,
+            mem_block: mem_ptr,
+            mem_size: mem_size,
+            write_pos: 0,
+            label_addrs: Vec::new(),
+            label_names: Vec::new(),
+            label_refs: Vec::new(),
+            current_aligned_write_pos: ALIGNED_WRITE_POSITION_NONE,
+            page_size: 4096,
+            dropped_bytes: false
+        };
```

Comment:
Just for simplicity, I think we could always include this field, but just not append any comments to the vector when asm_comments is not enabled.

Reply:
Okay. I'll change it back.